manifests/plugin/logs.pp

```diff
@@ -0,0 +1,28 @@
+# = Foreman Proxy Abrt plugin
+#
+# This class configures Logs buffer module.
+#
+# === Parameters:
+#
+# $enabled::                    Enables/disables the plugin
+#                               type:boolean
+#
+# $listen_on::                  Proxy feature listens on http, https, or both
+#
+class foreman_proxy::plugin::logs (
+  $enabled                 = $::foreman_proxy::plugin::logs::params::enabled,
+  $listen_on               = $::foreman_proxy::plugin::logs::params::listen_on,
+) inherits foreman_proxy::plugin::logs::params {
+
+  validate_bool($enabled)
+  validate_listen_on($listen_on)
+
+  foreman_proxy::plugin { 'logs':
+  } ->
+  foreman_proxy::settings_file { 'logs':
```

Comment:
I think we should refactor the plugin definition to include a config file. I think you risk not restarting the service this way.